specs/http-api/README.md

```diff
@@ -196,34 +208,65 @@ No authorization required. Offerings are publicly accessible
 # Exchanges
 An exchange is a series of linked tbDEX messages between Alice and a PFI for a single exchange. An exchange can be created by submitting an RFQ.
 
-## Submit RFQ
+## Create Exchange
 
 ### Endpoint
-`POST /exchanges/:exchange_id/rfq`
+`POST /exchanges/:exchange_id`
```

Comment:
We could take it a step further. `exchange_id` is already in the RFQ. It's superfluous in the URL.

```suggestion
`POST /exchanges`
```

Reply:
+1 to this - could this actually be applied to all our POST /exchanges/xyz endpoints, since all tbdex message metadata contain exchangeId?

Reply:
I'm a fan of removing it, @mistermoe any opposition?

Reply:
Going to open a separate issue for this because it'll require refactoring httpclient/httpserver in JS/KT outside the context of implementing callbacks